src/ert/scheduler/scheduler.py

```diff
@@ -125,13 +125,15 @@ async def cancel_all_jobs(self) -> None:
 
     async def _cancel_job_tasks(self) -> None:
         for task in self._job_tasks.values():
-            task.cancel()
-        with suppress(asyncio.TimeoutError):
-            await asyncio.wait(
-                self._job_tasks.values(),
-                timeout=1.0,
-                return_when=asyncio.ALL_COMPLETED,
-            )
+            if not task.done():
+                task.cancel()
+        _, pending = await asyncio.wait(
+            self._job_tasks.values(),
+            timeout=1.0,
+            return_when=asyncio.ALL_COMPLETED,
```

Comment:
When we use `return_when=asyncio.ALL_COMPLETED`, will any jobs be put in `pending`?

Reply:
If there is timeout then yes